src/scripts/configure.sh

```diff
@@ -78,3 +78,45 @@ then
 else
     update-grub
 fi
+
+# Prepare recovery partition, if it exists
+if [ -d "/boot/efi" -a -d "/cdrom" -a -d "/recovery" ]
```

Comment:
Here lies the meat of the modification. Copying the files from the cdrom if a recovery partition is mounted, creating a configuration file, and then setting up a boot entry